### ISSUE TITLE
Demuxer unable to see audio in mpeg-ts

### ISSUE THREAD

**Issue by anikzad** [google/shaka-packager#202]
Hi,
I've been trying to use the packager to convert a (any) mpeg-ts to fMP4. It seems the packager is able to see the video pid/track but never the audio for the input file which is an mpeg-ts format.  for example the output of the dump_info shows

>packager input=NFL_1800_short.mpg --dump_stream_info
[0101/001329:INFO:demuxer.cc(58)] Initialize Demuxer for file 'NFL_1800_short.mpg'.

File "NFL_1800_short.mpg":
Found 1 stream(s).
Stream [0] type: Video
 codec_string: avc1.420028
 time_scale: 90000
 duration: 9223372036854775807 (102481911520608.6 seconds)
 is_encrypted: false
 codec: H264
 width: 416
 height: 224
 pixel_aspect_ratio: 1:1
 trick_play_rate: 0
 nalu_length_size: 4

Packaging completed successfully.

whereas in reality the content has 
> tsinfo ../../NFL_1800_short.mpg
Reading from ../../NFL_1800_short.mpg
Scanning 10000 TS packets

Packet 257 is PAT
Program list:
    Program 1 -> PID 0055 (85)

Packet 260 is PMT with PID 0055 (85)
  Program 1, version 0, PCR PID 0013 (19)
  Program streams:
    PID 0011 (  17) -> Stream type 1b ( 27) H.264/14496-10 video (MPEG-4/AVC)
    PID 0012 (  18) -> Stream type 11 ( 17) 14496-3 Audio with LATM transport syntax (14496-3/AMD 1)

Found 16 PAT packets and 8 PMT packets in 10000 TS packets

is this a known bug? or did something?

**Comment by kqyang** [comment]
No, audio is supported. What is the audio codec used? Can you attach a sample content with this problem? Or email a sample content to me. You can find my email address in https://github.com/google/shaka-packager/blob/master/CONTRIBUTORS.

**Comment by anikzad** [comment]
I will email a short version of the stream which is about (40000*188) bytes. The audio/video info for this particular example are

Media Probe:
MPEG2-TS[SELF-INDEX:4:1] 0:00.38 1935Kbps

0x13:PCR 0x55:PMT 0x11:video(H.264 Baseline 4.0 0Kbps 416x224)
0x12:audio(AAC+ 48000:2:16:67)

Also note that the dump info reports the duration of the stream incorrectly as 102481911520608.6 seconds where as it's really 38.x seconds.

**Comment by kqyang** [comment]
@anikzad Thanks for the content.

We don't support audio with LATM transport syntax (0x11) yet. Only audio with ADTS transport syntax is supported (0x0F).

**Comment by kqyang** [comment]
To support it, we'll need to implement a latm parser similar to this one: https://github.com/google/shaka-packager/blob/master/packager/media/formats/mp2t/es_parser_adts.cc. Unfortunately, we don't have resource to work on it in the near future.
